platformLibs/src/platform/osx/ColorSync.def

```diff
@@ -0,0 +1,6 @@
+depends = CoreFoundation darwin posix
+language = Objective-C
+package = platform.ColorSync
+modules = ColorSync
+compilerOpts = -framework ColorSync
+linkerOpts = -framework ColorSync
```

Comment:
Isn't it a part of `ApplicationServices` platform library at the moment?

Reply:
🤦‍♂️

Reply:
Looks like we should remove it from `ApplicationServices` to make things simpler, but it will be a breaking change.

Reply:
Well, things are complicated here.
`ApplicationServices` imports the entire `ColorSync` (and even has the symlink to it under `Frameworks/`).
Anyway, it is a breaking change of course, so we shouldn't do this at the moment anyway.

Reply:
Ok, I'll disable it.